apps/src/code-studio/pd/workshops/components/EnrollInWorkshop.tsx

```diff
@@ -50,8 +50,8 @@ const EnrollInWorkshop: React.FC<EnrollInWorkshopProps> = ({
 }) => {
   const {handleClick, isSubmitting, alertState, setAlertState} =
     useWorkshopEnrollment({
-      workshopId: id,
-      userInfo,
+      workshop_id: id,
+      user_id: userInfo?.id,
```

Comment:
I updated the casing here to snake casing to match the `regional_partner_name` so we didn't have as much mixed casing here. I could have updated `regional_partner_name` but that would have required updates in more places. But I'm open to pushback if this doesn't seem necessary or we should go the other way (i.e. all camel casing).